website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -12,23 +12,48 @@ displayed_sidebar: "docs"
 - [Cloud upgrade guide](/docs/dbt-versions/upgrade-dbt-version-in-cloud)
```

Comment:
can we clarify it's 'dbt Core CLI installation guide' instead of 'CLI installation guide'?

Reply:
Will change in a follow up because this is copy/paste from all migration guides and we'll need to update all simultaneously.